lib/Models/Newsletter.php

```diff
@@ -411,8 +411,16 @@ function duplicate($data = []) {
       $options = NewsletterOption::where('newsletter_id', $this->id)
         ->findMany();
 
+      $ignored_option_field_ids = [
+        1, // isScheduled
```

Comment:
Please use option names instead of ids. Ids are autogenerated values so names should be safer.